packages/valory/skills/liquidity_trader_abci/rounds.py

```diff
@@ -495,6 +567,20 @@ class LiquidityTraderAbciApp(AbciApp[Event]):
             Event.SETTLE: FinishedTxPreparationRound,
             Event.UPDATE: DecisionMakingRound,
         },
+        DecideAgentStartingRound: {
+            Event.DONT_MOVE_TO_NEXT_AGENT: CallCheckpointRound,
+            Event.MOVE_TO_NEXT_AGENT: SwitchAgentStartingRound,
+            Event.DONE: CallCheckpointRound,
+            Event.NONE: CallCheckpointRound,
+            Event.NO_MAJORITY: CallCheckpointRound,
+        },
+        DecideAgentEndingRound: {
+            Event.DONT_MOVE_TO_NEXT_AGENT: PostTxSettlementRound,
+            Event.MOVE_TO_NEXT_AGENT: SwitchAgentEndingRound,
+            Event.DONE: PostTxSettlementRound,
+            Event.NONE: PostTxSettlementRound,
+            Event.NO_MAJORITY: PostTxSettlementRound,
+        },
```

Comment:
`ROUND_TIMEOUT` event is not mapped anywhere.

Reply:
ROUND_TIMEOUT mapped

Reply:
mapped all